### ISSUE TITLE
Permissions: Add "user" option

### ISSUE THREAD

**Issue by benhaynes** [directus/api#510]
We have `mine` which is specific to the current user, but I wonder if we should have an alias called `user` that does the same thing. I only say this because many Admins are using this to limit permissions to specific items that _they_ create.

eg: if an admin creates an item and "assigns" it to a user within `user_created` field type... then perhaps `user` permission and `user_owner` field type makes more sense?

Thoughts?

**Comment by WellingGuzman** [comment]
Ideally this should've been called `owner` instead of `user_created`. But we wanted to force to only be the user that requests it, so the meaning got lost in development.

Ideally it could be make more sense to check for permission on a field that means owner instead of author/creator. Example, An user can belongs a project now, but it can be transfer to another user (updating the field value), but the creator/author of that item could be any user.

In short I believe this should be implemented to attach a permission to an `owner` instead of a creator. maybe new type, or interface option? I don't want to remove this, but if we go to a new type for `owner`, `user_created` should be replaced as the field that holds the owner of an item.

**Comment by benhaynes** [comment]
That makes sense!

Still, I think we should plan on doing this in a new "dynamic rules" permissions system since that will give us a LOT of control. I will close this as a duplicate of the following ticket:

https://github.com/directus/api/issues/511
